[2023-03-29 16:55:36,338] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: postgres_22222222222.log_suc 2023-03-29T16:55:29.551939+00:00 [queued]>
[2023-03-29 16:55:36,360] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: postgres_22222222222.log_suc 2023-03-29T16:55:29.551939+00:00 [queued]>
[2023-03-29 16:55:36,362] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 16:55:36,364] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2023-03-29 16:55:36,367] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 16:55:36,396] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): log_suc> on 2023-03-29T16:55:29.551939+00:00
[2023-03-29 16:55:36,408] {standard_task_runner.py:52} INFO - Started process 28373 to run task
[2023-03-29 16:55:36,420] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgres_22222222222', 'log_suc', '2023-03-29T16:55:29.551939+00:00', '--job-id', '444', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/ETL.py', '--cfg-path', '/tmp/tmpgqc6_wod', '--error-file', '/tmp/tmpwu8k2gc3']
[2023-03-29 16:55:36,422] {standard_task_runner.py:77} INFO - Job 444: Subtask log_suc
[2023-03-29 16:55:36,491] {logging_mixin.py:109} INFO - Running <TaskInstance: postgres_22222222222.log_suc 2023-03-29T16:55:29.551939+00:00 [running]> on host 0bc3da97cebb
[2023-03-29 16:55:36,559] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgres_22222222222
AIRFLOW_CTX_TASK_ID=log_suc
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T16:55:29.551939+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T16:55:29.551939+00:00
[2023-03-29 16:55:36,590] {python.py:151} INFO - Done. Returned value was: None
[2023-03-29 16:55:36,613] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=postgres_22222222222, task_id=log_suc, execution_date=20230329T165529, start_date=20230329T165536, end_date=20230329T165536
[2023-03-29 16:55:36,680] {local_task_job.py:151} INFO - Task exited with return code 0
[2023-03-29 16:55:36,727] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
